platform/nodejs/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "build": "node build.js",
     "lint": "eslint js/ *.js",
-    "test": "mocha tests"
+    "test": "node test.js --mocha"
```

Comment:
If you don't like the Mocha tests or if they are causing any trouble, now it's possible to disable them by removing `--mocha` from here.

Reply:
I sure like to be able to run tests _finally_, see <https://github.com/NanoAdblocker/NanoCore/issues/77#issuecomment-629871500>.

Reply:
Thanks.

I wasn't sure you'd like Mocha, and in case you don't you can always choose a different framework or write one of your own.